pkgs/applications/networking/remote/vmware-horizon-client/default.nix

```diff
@@ -0,0 +1,69 @@
+{ stdenv, buildFHSUserEnv, fetchurl, makeWrapper, makeDesktopItem, libxslt, atk
+, fontconfig, freetype, gdk-pixbuf, glib, gtk2, libudev0-shim, libxml2
+, pango, pixman, libX11, libXext, libXinerama, libXrandr , libXrender
+, libXtst, libXcursor, libXi, libxkbfile , libXScrnSaver, zlib, liberation_ttf
+, libtiff, dbus, at-spi2-atk, harfbuzz, gtk3-x11, libuuid, pcsclite
+}:
+
+let
+  version = "2006";
+
+  sysArch =
+    if stdenv.hostPlatform.system == "x86_64-linux" then "x64"
+    else throw "Unsupported system: ${stdenv.hostPlatform.system}";
+    # The downloaded archive also contains i386 and ARM binaries, but these have not been tested.
+
+  vmwareHorizonClientFiles = stdenv.mkDerivation {
+    name = "vmwareHorizonClientFiles";
+    inherit version;
+    src = fetchurl {
+      url = https://download3.vmware.com/software/view/viewclients/CART21FQ2/vmware-view-client-linux-2006-8.0.0-16522670.tar.gz;
+      sha256 = "8c46d49fea42f8c1f7cf32a5f038f5a47d2b304743b1e4f4c68c658621b0e79c";
+    };
+    buildInputs = [ makeWrapper ];
+    installPhase = ''
+      mkdir ext $out
+      find ${sysArch} -type f -print0 | xargs -0n1 tar -Cext --strip-components=1 -xf
+      mv ext/bin ext/lib ext/share "$out"/
+
+      rm -f "$out/lib/vmware/gcc/libstdc++.so.6"
+
+      makeWrapper "$out/bin/vmware-view" "$out/bin/vmware-view_wrapper" \
+          --set GTK_THEME Adwaita \
```

Comment:
Why is this required? If possible could you link a public bug report or some other documentation of this issue? Or at least a comment?

Reply:
I added comments describing why the two workarounds are present.

The library deletion is because the launch script tries to use its own bundled version of libstdc++ if it does not detect a new enough version already present. Frankly, the version check is a little hacky, and it's distro-specific and fails on NixOS. The bundled version causes issues, and ours works fine.

As for the theme issue, basically its pretty easy to screw up the UI by changing themes, even Adwaita-dark: https://communities.vmware.com/thread/623875